contrib/debian/rules

```diff
@@ -11,7 +11,7 @@ ifeq "$(DEB_HOST_ARCH_BITS)" "32"
 endif
 
 ifneq ($(CI),)
-	export CI=--werror --wrap-mode=default
+	export CI=--werror --wrap-mode=default -Dgusb:tests=false
```

Comment:
this line can be dropped now since we're just skipping packaging when subprojects are involved